### PR TITLE
Mark CTI as deprecated

### LINKED ISSUE #144: Input file page should make clear that cti is deprecated
Was just browsing the website to try an answer a question for someone, and noticed that the input file page (https://cantera.org/tutorials/input-files.html) does not anywhere explicitly state that cti is deprecated.  

Probably some type of intro text to say "Cantera's input files, starting with Cantera 2.5, are based on the yaml format.  The previously used cti and xml formats have been discontinued, and are deprecated, beginning with Cantear 2.5..." would be nice.

Happy to work this up--just leaving this issue here as a note to remind myself to come back to it.

Also, while I'm on the page, I'm noticing the text `The first option will suffice for a majority of Cantera users` referring to one of the pre-packaged yaml files that ships with Cantera.  I'm not certain that this is true, or even what we intend (given the over-reliance on `gri30.yaml`).  I would recommend either just dropping the statement, or rather replacing it with some reference to converting from CK format (even though it is only strictly true for combustion-related users, it likely remains true that a majority of users will find converting from CK to be sufficient).

### LINKED ISSUE #145: Blowers Masel links are broken
In the YAML tutorial, the links to the Blowers Masel sections are broken: https://cantera.org/tutorials/yaml/reactions.html#id1

image

### PR DESCRIPTION
**Pull request by ischoegl**
Add clarifications that CTI is deprecated / replaced by YAML.

Closes #144, closes #145

### CODE REVIEW

**Comment by speth** [comment]
We already declared the CTI and XML formats as deprecated in Cantera 2.5. This is included in numerous places in the documentation, though obviously not everywhere. For the sake of a smoother transition, the idea was to start making noisy deprecation warnings only in Cantera 2.6, followed by removal in 3.0.

**Review by bryanwweber — APPROVED** [review]

**Comment by ischoegl** [comment]
@speth ... thanks for the clarification.

Before this gets merged, I'd like to make sure that the links I set are correct (the slugs go across different folders, and I'm not 100% sure that my syntax is correct).

**Comment by ischoegl** [comment]
:tada: ... had to update some file names to avoid clashing documentation links, but the YAML links now work as intended.

As a fun side-effect, I generated artifacts of the website build, so we don't have to build things locally: just navigate to 'Actions', select the latest build, and you'll see a downloadable 'artifact` that contains the website. I set it to be deleted after 5 days ...

From my side this is ready for review and/or merge.

**Comment by ischoegl** [comment]
Ended up fixing #145 as well as the YAML reference link, where docs for 2.5 pointed to the development version (77b8f0d9ac6191050e0b237e950e4f9f764acba)

File: pages/tutorials/input-files.rst [hunk 1]
```diff
@@ -16,19 +16,24 @@
 
 The required input files can be provided via one of several methods:
 
-- Use one of the pre-existing input files provided with Cantera
+- Use one of the pre-existing input files that are distributed with Cantera (note that
+  these input files are provided for convenience, and may not be suited for research)
```
Comment (decaluwe) [inline pages/tutorials/input-files.rst:20]: 😍

**Review by decaluwe — APPROVED** [review]
Looks great - thanks so much!

**Pull request merged**